code401challenges/src/test/java/code401challenges/linkedList/LinkedListTest.java

```diff
@@ -99,4 +99,15 @@
 
         assertEquals(expected, actual);
     }
+
+    @Test public void testKthFromEnd() {
```

Comment:
currently this test fails. Either reduce 6 so it's in range, or change the top to expect an error.
`@Test(expected = IndexOutOfBoundsException.class)`
Example here; https://www.baeldung.com/junit-assert-exception